plugins/inputs/win_eventlog/zsyscall_windows.go

```diff
@@ -94,8 +94,8 @@ func _EvtSubscribe(
 		8,
 		uintptr(session),
 		uintptr(signalEvent),
-		uintptr(unsafe.Pointer(channelPath)),
-		uintptr(unsafe.Pointer(query)),
+		uintptr(unsafe.Pointer(channelPath)), //nolint:gosec // G103: Use of unsafe calls should be audited
+		uintptr(unsafe.Pointer(query)),       //nolint:gosec // G103: Use of unsafe calls should be audited
```

Comment:
We should either be sure it is a valid use or replace it. Enabling the rule and then simply silencing all occurrences does not help IMO.

Reply:
From my perspective, for windows plugins which talk with system libraries, all these `unsafe...` are needed. I made few changes for linux (which could be substituted by something different than `unsafe...` call).

Reply:
@srebhan Check current version :)